### ISSUE TITLE
Week 13 Update

### ISSUE THREAD

**Issue by yiyangzhang30** [TongxinWang0405/BMI702_project#11]
Link to commit: 
https://github.com/TongxinWang0405/BMI702_project/commit/af5debde9a1ff0fa725468749f6a8996f7ab87ee

What we did:

- [x] Test on BERT text encoder and get results
- [x] Finalize all evaluation (on both holdout dataset and test split) and make tables and plots
- [x] Confirm that LLM generated caption training perform worse than normal prompts training

Future directions:

- [ ] Use more data set for training to improve performance/generalization
- [ ] Do hyperparameter tuning on all training set to help conclude on one model that generally perform well on all tasks
- [ ] Test on more various types of labels
